minimalkv/net/s3fsstore.py

```diff
@@ -66,11 +68,12 @@ def _create_filesystem(self) -> "S3FileSystem":
         if not has_s3fs:
             raise ImportError("Cannot find optional dependency s3fs.")
 
-        if "127.0.0.1" in self.endpoint_url:
+        if self.endpoint_url:
```

Comment:
This line made no sense to me and I need to pass a custom endpoint URL to the filesystem.
TOOD: Move into separate PR.

Reply:
https://github.com/data-engineering-collective/minimalkv/pull/81